src/sentry/services/hybrid_cloud/notifications/impl.py

```diff
@@ -87,6 +88,7 @@ def get_settings_for_recipient_by_parent(
     ) -> List[RpcNotificationSetting]:
         team_ids = [r.id for r in recipients if r.actor_type == ActorType.TEAM]
         user_ids = [r.id for r in recipients if r.actor_type == ActorType.USER]
+        # TODO(actorid) In tests recipients is sometimes a User. This needs to be solved.
```

Comment:
Which tests pass in users as recipients to `get_settings_for_recipient_by_parent`?
+1 for great use of `TODO`s and doc strings in the PR.

Reply:
Took another look, and this comment isn't correct. This method appears to be fine upon closer review.